en-US.json

```diff
@@ -955,10 +955,10 @@
 
 	"FAQ_04_H1": "How does Stremio compare to Flipboard?",
 	"FAQ_04_P1": "We don’t get that question that often, except when talking to start-up people :)",
-	"FAQ_04_P2": "Stremio is by design a content aggregator, which essentially makes it Flipboard for video. This is very apparent in our Board section, which gives you catalogues and new episode/new video notifications for what you follow, just like Flipboard.",
+	"FAQ_04_P2": "Stremio is by design a content aggregator, which essentially makes it Flipboard for video. This is very apparent in our Board section, which gives you catalogs and new episode/new video notifications for what you follow, just like Flipboard.",
 
 	"FAQ_05_H1": "How does Stremio compare to Popcorn Time?",
-	"FAQ_05_P1": "Stremio and Popcorn Time are similar in user interface, because there aren’t many ways to do video entertainment catalogues right :)",
+	"FAQ_05_P1": "Stremio and Popcorn Time are similar in user interface, because there aren’t many ways to do video entertainment catalogs right :)",
```

Comment:
We use "catalogue" intentionally everywhere, see: https://www.grammarly.com/blog/catalog-catalogue/